components/Table/Table.js

```diff
@@ -10,21 +10,39 @@ export class Table {
   }
 
   createTable() {
-    this.thead = document.createElement("thead");
+    this.thead = document.createElement("div");
```

Comment:
Since we use `div` instead of the table and we do not have `thead` element, let's use a more descriptive name
Let's call it `headerEl`